rllib/utils/tests/test_actor_manager.py

```diff
@@ -170,7 +171,7 @@ def test_sync_call_same_actor_multiple_times(self):
         results = manager.foreach_actor(
             lambda w: w.call(),
             healthy_only=False,
-            remote_actor_ids=[0, 0],
+            remote_actor_ids=[1, 1],
```

Comment:
Is there any way that we can make local_worker index separate from remote_worker indices? I don't like the fact that remote_actor_ids have to start from 1. It's so confusing.

Reply:
used to be, they live in the same array ...
I think Avnish will get rid of the concept of local worker if he pulls off his change.

Reply:
For WorkerSet, let's not change this now as this would probably break a ton of things. But yes, for ActorManager, let's do this right from the start.